client/js/components/OnboardingThree/OnboardingThree.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import { styles } from "./styles";
+const OnboardingThree = ({ navigation }) => {
+  const handleReisterClick = () => {
+    navigation.navigate("Layout");
+  };
+  const handleSignInClick = () => {
+    navigation.navigate("Layout");
+  };
```

Comment:
Both of these functions have the same logic. Please make just one function to help you navigate to Layout.